ui/shared/AssetSelector.qml

```diff
@@ -7,9 +7,7 @@ import "../imports"
 Item {
     id: root
     property var assets
-    property var selectedAsset: {
-        "symbol": "snt", "name": "", "value": "", "fiatValue": "", "address": ""
-    }
```

Comment:
@emizzle this initialization is was caused the AssetSelector to emit a `selectedAsset` with an empty `value` (which caused the amount input to break).

I think it's fine to not have it initialized